lib/tree-view.coffee

```diff
@@ -573,12 +573,23 @@ class TreeView extends View
             if repo = repoForPath(selectedPath)
               repo.getPathStatus(selectedPath)
           if failedDeletions.length > 0
-            atom.notifications.addError "The following #{if failedDeletions.length > 1 then 'files' else 'file'} couldn't be moved to trash#{if process.platform is 'linux' then " (is `gvfs-trash` installed?)" else ""}",
+            atom.notifications.addError @formatTrashFailureMessage(failedDeletions),
               detail: "#{failedDeletions.join('\n')}"
               dismissable: true
           @updateRoots()
         "Cancel": null
 
+  formatTrashFailureMessage: (failedDeletions) ->
+    fileText = if failedDeletions.length > 1 then 'files' else 'file'
+
+    "The following #{fileText} couldn't be moved to the trash. #{@formatTrashEnabledMessage()}"
+
+  formatTrashEnabledMessage: ->
+    switch process.platform
+      when 'linux' then 'Is `gvfs-trash` installed?'
+      when 'darwin' then 'Is Trash enabled on the volume where the files are stored?'
+      when 'win32' then 'Is RecycleBin support enabled on the drive where the files are stored?'
```

Comment:
Recycle Bin should be two words.  An alternative wording could also be `Is there a Recycle Bin on the drive where the files are stored?`